lib/ngx/balancer.lua

```diff
@@ -101,4 +104,23 @@ function _M.get_last_failure()
 end
 
 
+function _M.set_timeout(connect_timeout, send_timeout, read_timeout)
```

Comment:
Maybe it's more flexible to provide separate `set_xxx_timeout` methods? Sometimes we may just want to adjust a single timeout entry, for example.